pkg/debug/ssh/sshd_config

```diff
@@ -0,0 +1,5 @@
+AuthorizedKeysFile .ssh/authorized_keys
+AllowTcpForwarding yes
+GatewayPorts no
+X11Forwarding no
+Subsystem sftp /usr/lib/ssh/sftp-server
```

Comment:
Oh, cool; it will also solve the issue with the SCP in the latest distros: we had to use an extra `-O` flag to make it work with our sshd config.
See EV-877

Reply:
@OhmSpectator , what was the problem before? Only version?

Reply:
From the Issue description:
```
The SCP protocol has been deprecated in OpenSSH 9.0, which is included in Apple's Ventura OS and is also being rolled out in various Linux distributions. The scp utility now requires the -O flag to maintain backward compatibility.

To ensure compatibility and embrace more modern file transfer protocols, it is proposed to enable SFTP (SSH File Transfer Protocol) support in EVE OS's SSH Daemon (sshd).
```